app/src/main/java/org/mozilla/rocket/incognito/IncognitoCore.kt

```diff
@@ -0,0 +1,16 @@
+package org.mozilla.rocket.incognito
+
+import android.content.Context
+import android.preference.PreferenceManager
+
+class IncognitoCore {
+    companion object {
+        const val PREF_KEY_INCOGNITO_ENABLED = "pref_key_incognito_enabled"
+
+        private const val PREF_KEY_INCOGNITO_ENABLED_DEFAULT = false
```

Comment:
should we set default value in xml file?

https://developer.android.com/guide/topics/ui/settings#Defaults